Modules/_lzmamodule.c

```diff
@@ -23,6 +23,20 @@
     } } while (0)
 #define RELEASE_LOCK(obj) PyThread_release_lock((obj)->lock)
 
+typedef struct {
+    PyTypeObject *lzma_compressor_type;
+    PyTypeObject *lzma_decompressor_type;
+    PyObject *error_type;
```

Comment:
I suggest to rename to "error" to make the code shorter, and because it was previously called "Error" (and not "Error_type").